ewatercycle/config/ewatercycle.yaml

```
@@ -1,7 +1,7 @@
 esmvaltool_config: null
 grdc_location: null
-container_engine: null
-singularity_dir: null
-output_dir: null
-wflow.singularity_image: null
-wflow.docker_image: null
+container_engine: docker
+singularity_dir: './'
+output_dir: './'
+wflow.singularity_image: "./ewatercycle-wflow-grpc4bmi.sif"
```

Not sure if a file is a good default, could lead to unexpected errors. We could use the Docker Hub as well for singularity.
```suggestion
wflow.singularity_image: "docker://ewatercycle/wflow-grpc4bmi:latest"
```